meson.build

```diff
@@ -44,6 +44,37 @@ if get_option('enable-tflite')
   cdata.set('HAVE_TFLITE', 1)
 endif
 
+# Define library dependencies for TensorRT support
+if get_option('enable-tensorrt')
+
+  # CUDA versions supported
+  cuda_versions = [
+    '10.0',
+  ]
+
+  # Find CUDA version
+  cuda_ver = ''
+  foreach v : cuda_versions
+    if cuda_ver == ''
```

Comment:
I'd break the loop instead of asking for this. This way we don't loop unnecessarily.